src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariants.java

```diff
@@ -198,6 +200,13 @@
                 return new String[]{"No default architecture for tensor type:" + tensorType.name()};
             }
         }
+
+        IntelGKLUtils utils = new IntelGKLUtils();
+        if (utils.isAvxSupported() == false)
+        {
+            return new String[]{CNNScoreVariants.AVXREQUIRED_ERROR};
```

Comment:
Per the conversation last time, we can't just completely fail to run when AVX isn't present; we need to allow the user to run using the alternative conda environment. Given that, if we were to make Intel-TF the default, what does the failure mode look like when AVX isn't present ? IIRC, it was a pretty hard failure ? If there were a deterministic exception raised in Python it might be possible detect that and handle it gracefully, and change this message to tell the user what to do.

Reply:
Maybe the answer is for the conda environments to set an extra environment variable that would allow GATK to detect which conda environment it's in. Then you could have a check in `CNNScoreVariants` that aborts the tool only if AVX is not present AND you're running in the Intel conda environment, and point the user to the non-Intel conda environment in the error message.